Purchases/Networking/SubscribersAPI.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  SubscribersAPI.swift
+//
+//  Created by Joshua Liebowitz on 11/17/21.
+
+import Foundation
+
+class SubscribersAPI {
```

Comment:
This was just my way of extracting out a number of methods that share things like callback caches. Eventually, this could be merged back into Backend.swift as the remaining items there will be split out into their own operations as well. 
Better yet, instead of having just ONE Backend object to handle all this stuff, we can create a number of them that handle specific groups of tasks.